src/styles/index.css

```diff
@@ -0,0 +1,45 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+  background-color: rgb(86, 137, 228);
+}
+
+main {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+}
+
+img:hover {
+  opacity: 0.5;
+}
+
```

Comment:
- [ ] Please try to add nice-looking styles to your `details` page so as to improve the User experience. You can do this by **reducing the image sizes and using a good layout to align them very well in a good-looking style** 😎
This is a capstone project and you will put it on your portfolio, therefore It should look pleasing to people who visit your portfolio 😉